charts/descheduler/README.md

```diff
@@ -69,6 +69,8 @@ The following table lists the configurable parameters of the _descheduler_ chart
 | `serviceAccount.create`             | If `true`, create a service account for the cron job                                                                  | `true`                               |
 | `serviceAccount.name`               | The name of the service account to use, if not set and create is true a name is generated using the fullname template | `nil`                                |
 | `serviceAccount.annotations`        | Specifies custom annotations for the serviceAccount                                                                   | `{}`                                 |
+| `podAnnotations`                    | Annotations to add to Pods                                                                                            | `{}`                                 |
+| `podLabels`                         | Labels to add to Pods                                                                                                 | `{}`                                 |
```

Comment:
maybe clarify that this is added to the descheduler Pods?

Reply:
Done.